app/script/user/UserRepository.js

```diff
@@ -568,18 +568,21 @@ z.user.UserRepository = class UserRepository {
     });
   }
 
-  changeAvailability(availability, method) {
+  setAvailability(availability, method) {
```

Comment:
The parameter name `method` needs to become a little more declarative (just saw it now 🙈 ).

Reply:
Yeah, Any suggestions? Just used the name that was specified in the event we use this in.